cebra/data/single_session.py

```diff
@@ -351,17 +359,55 @@ def __post_init__(self):
         #            e.g. integrating the FAISS dataloader back in.
         super().__post_init__()
 
-        if self.conditional != "time_delta":
-            raise NotImplementedError(
-                "Hybrid training is currently only implemented using the ``time_delta`` "
-                "continual distribution.")
-
-        self.time_distribution = cebra.distributions.TimeContrastive(
-            time_offset=self.time_offset,
-            num_samples=len(self.dataset.neural),
-            device=self.device)
-        self.behavior_distribution = cebra.distributions.TimedeltaDistribution(
-            self.dataset.continuous_index, self.time_offset, device=self.device)
+        # BEHAVIOR DISTRIBUTION
+
+        if self.conditional == "time":
+            self.behavior_distribution = cebra.distributions.TimeContrastive(
```

Comment:
why behavior as name?

Reply:
See comment above; not ideal but consistently with the rest of the codebase...

Reply:
#243